docs/docsite/rst/network/user_guide/network_debug_troubleshooting.rst

```diff
@@ -40,7 +40,7 @@ Enabling Networking logging and how to read the logfile
 
 Ansible includes logging to help diagnose and troubleshoot issues regarding Ansible Networking modules.
 
-Because logging is very verbose, it is disabled by default. It can be enabled with the :envvar:`ANSIBLE_LOG_PATH` and :envvar:`ANSIBLE_DEBUG` options on the ansible-controller, that is the machine running ``ansible-playbook``.
+Because logging is very verbose, it is disabled by default. It can be enabled with the :envvar:`ANSIBLE_LOG_PATH` and :envvar:`ANSIBLE_DEBUG` options on the ansible-control-node, that is the machine running ``ansible-playbook``.
```

Comment:
```suggestion
Because logging is very verbose, it is disabled by default. It can be enabled with the :envvar:`ANSIBLE_LOG_PATH` and :envvar:`ANSIBLE_DEBUG` options on the Ansible control node, that is the machine running ``ansible-playbook``.
```